.github/workflows/publish.yml

```diff
@@ -112,7 +121,7 @@ jobs:
     # Needs to be "-E" instead of "-r" on macOS
     - name: Replace version numbers in ios/README.md
       run: |
-        sed -i.bk -r "s/[0-9]+\.[0-9]+\.[0-9]+/$NEW_VERSION/g" ios/README.md
+        sed -i.bk -r "s/[0-9]+\.[0-9]+\.[0-9]+/$IOS_VERSION/g" ios/README.md
```

Comment:
Would you mind reverting this one back to `NEW_VERSION` or changing the rest of the entries in this file to `IOS_VERSION` and `ANDROID_VERSION` or maybe `NATIVE_VERSION` since both of those are kept in sync?